pkg/detectors/github/v1/github_old.go

```diff
@@ -74,6 +75,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 		s1.ExtraData = map[string]string{
 			"rotation_guide": "https://howtorotate.com/docs/tutorials/github/",
+			"version":        fmt.Sprintf("%d", s.Version()),
```

Comment:
Instead of this being a responsibility of individual detectors, what do you think about setting this generically in the engine? We could probably do the same thing for the `DetectorType` too.

Reply:
I was going to suggest this as well. It seems much easier and less error prone to modify engine + printers.